config/default/configmap/inferenceservice.yaml

```diff
@@ -18,7 +18,11 @@ data:
                "1.13.0",
                "1.13.0-gpu",
                "1.14.0",
-               "1.14.0-gpu"
+               "1.14.0-gpu",
+               "2.0.0",
+               "2.0.0-gpu",
+               "2.1.0",
+               "2.1.0-gpu",               
```

Comment:
Did you test all of these? Can you add e2e integration tests for this?

Reply:
Also, do you mind separating these PRs. One to fix env vars, the other to add new tensorflow versions?

Reply:
Ok, I've removed those changes for now, will refile under a separate PR. Are there any existing e2e tests I could build off of?

Reply:
https://github.com/kubeflow/kfserving/tree/master/test/e2e/predictor